tests/test_orbit.py

```diff
@@ -2174,6 +2174,11 @@ def test_orbit_setup():
     assert numpy.fabs(o.pmll(obs=obs)-0.5) < 10.**-10., 'Orbit pmll setup does not agree with o.pmll()'
     assert numpy.fabs(o.pmbb(obs=obs)-0.) < 10.**-10., 'Orbit pmbb setup does not agree with o.pmbb()'
     assert numpy.fabs(o.vlos(obs=obs)-30.) < 10.**-10., 'Orbit vlos setup does not agree with o.vlos()'
+    #test galactocentric spherical coordinates
+    o= Orbit([2.,2.**0.5,1.,2.,2.**0.5,0.5])
+    assert numpy.fabs(o.vr()-2.) < 10.**-10., 'Orbit galactocentric spherical coordinates are not correct'
+    assert numpy.fabs(o.vtheta()-0.) < 10.**-10., 'Orbit galactocentric spherical coordinates are not correct'
+    assert numpy.fabs(o.theta()-numpy.pi/4) < 10.**-10.,  'Orbit galactocentric spherical coordinates are not correct'
```

Comment:
Think you need to add a test of ``o.vr`` for a 2D orbit, because that's the one line that you added that isn't covered right now.

Reply:
I assume this is needed to test the clause where `self.dim < 3` orbits o.vr() returns vR. Added a test to `test_orbit_setup_linear`

Reply:
I would have tested it for a ``planarOrbit``, but this seems to do the trick!